zulip/integrations/google/doc.md

```diff
@@ -1,72 +1,172 @@
-Get Google Calendar reminders in Zulip! This is a great way to see
-your reminders directly in your Zulip feed.
+# Zulip Google Calendar integration
+
+Get Zulip notifications for your Google Calendar events!
+
+### Create Zulip bot for Google Calendar notifications
+
+{start_tabs}
+
+1.  {!create-a-generic-bot.md!}
 
 1.  {!download-python-bindings.md!}
 
-    This bot should be set up on a trusted machine, because your API
-    key is visible to local users through the command line or config
-    file.
+1.  [Download your bot's `zuliprc` file][download-zuliprc], and save it as
+    `.zuliprc` in your `~/` directory.
+
+[download-zuliprc]: /api/configuring-python-bindings#download-a-zuliprc-file
+
+1.  Optionally, to configure the bot, add a **google-calendar** section to
+    the `.zuliprc` file, like below:
+
+    ```
+    [google-calendar]
+    calendar=primary
+    interval=3600
+    tokens_path=/home/username/zulip/integrations/google-oauth-tokens.json
+    client_secret_path=/home/username/zulip/integrations/client-secret-credentials.json
+    channel=core-team
+    topic=scheduled events
+    noauth_local_webserver=
+    format_message=The event [{title}]({calendar_link}), at {location}, is from {start} to {end}.\n> {description}\n\n[Join call]({google_meet_link}).
```

Comment:
If the Google Meet link is missing from the payload, the join call link would be broken, correct? I wonder if we want that in the message template if so...